app/services/braze_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class BrazeService
+  class << self
+    def send_swa_my_collection_email(submission_id)
+      submission = Submission.find(submission_id)
+      raise 'Still processing images.' unless submission.ready?
+
+      raise 'User lacks email.' if submission.email.blank?
+
+      recipient = [
+        {
+          external_user_id: submission.user.gravity_user_id,
```

Comment:
We need to play more with `external_user_id` and `user_alias`, I think the gem is not supporting sending only the `user_alias`, so I believe this would support only cases where the user is already synched in Braze.

Reply:
Gravity keeps user accounts synched with Braze, so it is safe to assume that if a submission has a `gravity_user_id` that it is synched to Braze.